servers/src/main/webapp/worker/general.jsp

```diff
@@ -70,7 +69,7 @@
                 </tr>
                 <% List<Long> capacityBytesOnTiers = (List<Long>) request.getAttribute("capacityBytesOnTiers"); %>
                 <% List<Long> usedBytesOnTiers = (List<Long>) request.getAttribute("usedBytesOnTiers"); %>
-                <% for (int i = 0; i < StorageLevelAlias.SIZE; i ++) { %>
+                <% for (int i = 0; i < capacityBytesOnTiers.size(); i ++) { %>
```

Comment:
If user only set MEM and HDD without SSD, the UI will show capacity info of MEM and SSD.